### PR TITLE
Feature/pipdep 1068

### PR DESCRIPTION
**Pull request by bertranddemiddelaer**

### CODE REVIEW

File: .github/workflows/pre-merge.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - 'bugfix/**'
+      - 'w/**'
+      - 'q/**'
+
+
+jobs:
+  test-states-syntax:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
```
Comment (erwan-b) [inline .github/workflows/pre-merge.yml:19]: Use this action to freez the node version
https://github.com/actions/setup-node

File: .github/workflows/pre-merge.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      - name: install mocha
+        run: npm install mocha -g
```
Comment (erwan-b) [inline .github/workflows/pre-merge.yml]: This deps is the package.json, you should be able to remove it without trouble.

File: .github/workflows/pre-merge.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      - 'user/**'
+      - 'feature/**'
+      - 'improvement/**'
+      - 'bugfix/**'
+      - 'w/**'
+      - 'q/**'
+
+
+jobs:
+  test-states-syntax:
```
Comment (tcarmet) [inline .github/workflows/pre-merge.yml]:
```suggestion
  tests:
```

File: .github/workflows/pre-merge.yml (hunk 1 above)
Comment (tcarmet) [reply .github/workflows/pre-merge.yml:19]:
```suggestion
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: '10'
          cache: 'npm'
```

Comment (bertranddemiddelaer) [reply .github/workflows/pre-merge.yml:19]: cache: 'npm' broke the CI, as it asked for a package-llock.json or yarn.lock file I did not had. I removed it.

**Review by tcarmet — APPROVED** [review]

**Review by erwan-b — APPROVED** [review]

**Pull request merged**